### PR TITLE
Styling changing for #18

### PR DESCRIPTION
**Pull request by manez**
A bunch of formatting to address #18. Hopefully without destroying anything important...

### CODE REVIEW

File: Deployment Guides/Provisioning-Islandora-on-Ubuntu.md [hunk 1]
```diff
@@ -120,280 +129,279 @@ cat ~/islandora-install.properties
...
-<code>./configure</code>
+`./configure`
  
-<code>make && make install</code> 
+`make && make install`
 
+`gs --version` should return result.
 
-_should return result_  
+#### ffmpeg  <a id="ffmpeg"></a>
```
Comment (manez) [inline Deployment Guides/Provisioning-Islandora-on-Ubuntu.md:177]: This is in here twice - more detail on the one below. Is this intentional? Might need more explanation, if so.

File: Deployment Guides/Provisioning-Islandora-on-Ubuntu.md [hunk 2]
```diff
@@ -120,280 +129,279 @@ cat ~/islandora-install.properties
...
 
-<code>./configure --disable-shared</code>  
+`make`  
 
-<code>make</code>  
+`checkinstall --pkgname=libopus --pkgversion="$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default  `
 
-checkinstall --pkgname=libopus --pkgversion="$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default  
-</code>
+#### ffmpeg <a id="ffmpeg2"></a>
```
Comment (manez) [inline Deployment Guides/Provisioning-Islandora-on-Ubuntu.md:258]: The other ffmpeg section. Different from the first, but labeled the same. Deliberate or duplicate?

File: Deployment Guides/Provisioning-Islandora-on-Ubuntu.md [hunk 3]
```diff
@@ -1,67 +1,75 @@
...
 For general purpose repository sites, we recommend at minimum 2-4 CPU cores and 8GB-10GB of RAM.
 
 This would be scaled up according to user-load, the type of derivative files being generated by the system (ie. creating web friendly version of archival quality images or video are quite processor intensive) and many other factors.  
 
-### SSH keys
-User accounts e.g. islandora  
-Sudoers config (location, ideal setup)  
-iptables (turn off for initial deploy?)  
-Install Dependencies and Pre-setup   
+## SSH keys <a id="ssh-keys"></a>
```
Comment (manez) [inline Deployment Guides/Provisioning-Islandora-on-Ubuntu.md:64]: Isn't in the index, wasn't sure what it's for. Might need more explanatory text?

**Comment by manez** [comment]
Had a few questions. Might not be quite ready for merge.

**Comment by manez** [comment]
Sounds good to me! I just want a few more eyes on my work  to make sure i did ok, and to look at a few things I was not sure about.

File: Deployment Guides/Provisioning-Islandora-on-Ubuntu.md (hunk 1 above)
Comment (lutaylor) [inline Deployment Guides/Provisioning-Islandora-on-Ubuntu.md:177]: I think this was more of a markup typo on my part. Probably don't need that heading. The first part is just making a folder to put all the ffmpeg related dependancy stuff in. The second part is the actual ffmpeg build.

File: Deployment Guides/Provisioning-Islandora-on-Ubuntu.md (hunk 3 above)
Comment (lutaylor) [inline Deployment Guides/Provisioning-Islandora-on-Ubuntu.md:64]: Don't think we need that in this doc. It was just a filler spot that Gavin put in on stuff that could be filled out a bit more. Probably could take that out until someone draws up the actual details. Something like this should probably be opened as an issue for someone to fill out.

Comment (manez) [inline Deployment Guides/Provisioning-Islandora-on-Ubuntu.md:64]: Thanks! I took them out with PR #21 and put them in issue #22

**Pull request merged**